Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -465,14 +468,49 @@ void Wiimote::GetIRData(u8* const data, bool use_accel)
   static const double dist1 = 100.0 / camWidth;  // this seems the optimal distance for zelda
   static const double dist2 = 1.2 * dist1;
 
-  for (auto& vtx : v)
+  float aspect =
+      SConfig::GetInstance().m_SYSCONF->GetData<bool>("IPL.AR") ? 16.0 / 9.0 : 4.0 / 3.0;
+
+  float yScale = bpmem.triggerEFBCopy.scale_invert ? 256.0f / (float)bpmem.dispcopyyscale :
+                                                     (float)bpmem.dispcopyyscale / 256.0f;
+
+  u32 height = static_cast<u32>(1.0f + (float)bpmem.copyTexSrcWH.y * yScale);
```

Comment:
Are these values even guaranteed to be filled in at Wiimote poll time? They can be junk unless an active copy is going through them, and I'm quite sure GX will stomp on registers it doesn't care about when prepping other commands -- masking is expensive.